usb1/__pyinstaller/test_libusb1_packaging.py

```diff
@@ -0,0 +1,22 @@
+import subprocess
+from PyInstaller import __main__ as pyi_main
+
+
+def test_pyi_hooksample(tmp_path):

```

Comment:
How can I run this test myself ?

Reply:
The tests can be run using the following steps:

````
pip install pyinstaller[hook_testing]
python -m PyInstaller.utils.run_tests --include_only usb1
````

I will add these instructions directly to the module comment of the test file for future reference.